src/lib-dcrypt/ostream-encrypt.c

```diff
@@ -1,4 +1,4 @@
-/* file truct dcrypt_public_keyyntax
+/* file struct dcrypt_public_key syntax
```

Comment:
I'm really not certain about this correction. The corrupt content was present when it was introduced.
@cmouse: Did I guess correctly?